client/index.html

```diff
@@ -58,6 +58,7 @@ <h1 class="title">The Lounge is loading…</h1>
 				<div id="chat-container" class="window">
 					<div id="chat"></div>
 					<form id="form" method="post" action="">
+						<div id="connection-error"><span>Connection failed: <span id="connection-error-reason">??</span>. Refresh the page to reconnect.</span></div>
```

Comment:
> Nope. It's to correctly do the flexbox.

Really?? It's so weird to look at 😅.
Anyway, if you move it above the input as mentioned below, I think height should be normal text height + whatever margin, as it won't have to take up the input's height exactly.

> Specific messages absolutely help when you hit a certain issue.

I strongly disagree that `Connection failed: transport close` (which is what you hit every time 3G/Wifi connection is lost, i.e. 95% of the time this feature will be useful) is helpful to the user. What "failed", I didn't attempt any action? What transport? I didn't close anything?

I understand that from an engineering standpoint, it's interesting to know (which is why I'm perfectly OK to have it in the browser console), but from a UX standpoint, it's terrible. In fact, I don't know any general public app that displays anything else than `Connection lost`, `Connecting...`, `Please wait while you are being connected`, etc.
By making it more detailed, we make it confusing in most if not all cases that most users will experience.
This reminds me a discussion with my UX designer roommate who was telling me how his developers were always trying to display server messages, while what users really want to see is if it's working or not and what to do next when it's not :D

So let's please make it `Connection lost. Click here to reconnect` and be done with it. This will also simplify code of this PR, which is great if it's not meant to live forever :-)